server/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/Location.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.extendedapi;
+
+import lombok.Data;
+import org.eclipse.lsp4j.Position;
+
+/**
+ * Data transport object of CF AST.
+ */
+@Data
+public class Location {
```

Comment:
Why don't you use the `Location` from LSP4J?

Reply:
Unfortunately, the Location in CFAST Extension is a Range, and the Location in lsp4j has a range.